### PR TITLE
14-7 issue done

### PR DESCRIPTION
**Pull request by AlexanderVVasilenko**
check this gimnocode and leave a comment how i can impove it

### CODE REVIEW

**Review by Sanverik — CHANGES_REQUESTED** [review]
- Don’t have a task description
- Don’t have any test cases
- Don’t see any validation of input

File: Module14/07_years/main.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def print_years_with_3_same_digits(low_year, up_year):
+    for year in range(low_year, up_year + 1):
+        counter_1 = 1
+        counter_2 = 1
+        first_number = year % 10
```
Comment (Sanverik) [inline Module14/07_years/main.py]: maybe `first_digit`?

File: Module14/07_years/main.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            if year_ % 10 == first_number:
+                counter_1 += 1
+            elif year_ % 10 == second_number:
+                counter_2 += 1
+            year_ //= 10
+        if counter_1 >= 3 or counter_2 >= 3:
+            print(year)
+
+
+first_year = int(input("Enter the first year: "))
```
Comment (Sanverik) [inline Module14/07_years/main.py:25]: what would be if a random string is entered?

Comment (Sanverik) [reply Module14/07_years/main.py:25]: negative year?

File: Module14/07_years/main.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+def print_years_with_3_same_digits(low_year, up_year):
+    for year in range(low_year, up_year + 1):
```
Comment (Sanverik) [inline Module14/07_years/main.py:2]: What would be if the first parameter is greater or equal to the second?

File: Module14/07_years/main.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        counter_1 = 1
+        counter_2 = 1
```
Comment (Sanverik) [inline Module14/07_years/main.py]: Unclear meaning/usage of both counters by reading their names

File: Module14/07_years/main.py (hunk 3 above)
Comment (AlexanderVVasilenko) [reply Module14/07_years/main.py:2]: Error from for loop

File: Module14/07_years/main.py (hunk 2 above)
Comment (AlexanderVVasilenko) [reply Module14/07_years/main.py:25]: If you enter the not integer string you will get TypeError
Negative value is unforeseen by task

**Pull request merged**